library/varnish

```diff
@@ -0,0 +1,13 @@
+# this file was generated using https://github.com/varnish/docker-varnish/blob/8312c29a9a7dbd161c291683a77ac209fe7b3cd0/populate.sh
+Maintainers: Guillaume Quintard <guillaume@varni.sh> (@gquintard)
+GitRepo: https://github.com/varnish/docker-varnish.git
+
+Tags: 6.2, 6.2.0-1, 6.2.0, 6, latest, fresh
+Architectures: amd64
+Directory: fresh/debian
+GitCommit: 8312c29a9a7dbd161c291683a77ac209fe7b3cd0
```

Comment:
One more push to get these to https://github.com/varnish/docker-varnish/commit/82ba69727be4e78de5a3e07a63e8b62d92ab958d